app/scripts/lib/EntryBridge.ts

```diff
@@ -0,0 +1,201 @@
+/**
+ * The type of change that occurred.
+ */
+export type ChangeType = 'added' | 'updated' | 'deleted';
```

Comment:
This type is a great idea as it's likely the source updates will be very similar for add and update.

Though would it be cleaner to use an enum to define these supported values rather than a string union?

Reply:
Done in next update.